src/connector/src/connector_common/common.rs

```diff
@@ -46,6 +46,10 @@ pub const PRIVATE_LINK_TARGETS_KEY: &str = "privatelink.targets";
 
 const AWS_MSK_IAM_AUTH: &str = "AWS_MSK_IAM";
 
+/// The environment variable to disable using default credential from environment.
+/// It's recommended to set this variable to `false` in cloud hosting environment.
+const DISABLE_DEFAULT_CREDENTIAL: &str = "DISABLE_DEFAULT_CREDENTIAL";
```

Comment:
Shall it affect services from cloud vendors besides AWS as well?

Reply:
If the idea looks good to you, I'll add the support of Azure/GCP in future PRs. This PR was supposed to complete #17851

Reply:
I see. Thanks!